Jenkinsfile

```diff
@@ -106,18 +106,7 @@ pipeline {
                         junit_files: "*.xml non-exist*.xml",
                         failure_artifacts: env.STAGE_NAME
                 }
-                post {
-                    unsuccessful {
-                        scmNotify description: env.STAGE_NAME,
-                                  context: 'test/' + env.STAGE_NAME,
-                                  status: 'FAILURE'
-                    }
-                    success {
-                        scmNotify description: env.STAGE_NAME,
-                                  context: 'test/' + env.STAGE_NAME,
-                                  status: 'SUCCESS'
-                    }
-                }
+                // runTest handles SCM notification
```

Comment:
suggest change these comments to
// runTest handles SCM notification via stepResult

Reply:
Fixed in current push